### ISSUE TITLE
Implement array data structure and add forms for Hobbies

### ISSUE THREAD

**Issue by praveenscience** [CatsInTech/Rezume#17]
Let's discuss on a data structure for Hobbies and then let's implement this on #8.

**Comment by lakshyajit165** [comment]
@praveenscience. I would like to work on this issue.

**Comment by praveenscience** [comment]
Awesome @lakshyajit165, would you like to start with discussing on what you're planning to do in the main thread on #8?

**Comment by lakshyajit165** [comment]
@praveenscience sure. Could you allow me till late evening? I am working now and my shift ends around 9.p.m. that's why.

**Comment by praveenscience** [comment]
No worries at all bro! Let's discuss on #8 first. 😊

**Comment by lakshyajit165** [comment]
Thanks @praveenscience. Will get back to you ASAP.

**Comment by lakshyajit165** [comment]
@praveenscience, I have been thinking about a data structure for the hobbies and what I feel is that, a **hobby can have a name and a type**, so it should be an **array of objects**.
> For example, **"coding" as a hobby** can have its **name as "coding"** and **type as "learning(or Computer Science - to be a little generic)"** and similarly **"Music" as a hobby** can have a specific **name like (singing/playing an instrument)** and **type as "Arts"**.

**Comment by praveenscience** [comment]
Nice ideas, let's discuss at #8.

**Comment by utkarshnashine** [comment]
Sir,I would like to work on this issue as GSSOC'22 participant

**Comment by praveenscience** [comment]
Okay sir, tell what needs to be done @utkarshnashine? You're not even reading what needs to be done.

**Comment by utkarshnashine** [comment]
Sir actually I am a begineer and I want to contribute and I am just trying to run the project and get the essence so I can contribute and implement the data structure.Sorry for not replying

**Comment by praveenscience** [comment]
@utkarshnashine Sir, all I ask is, did you even read what's this about?

**Comment by utkarshnashine** [comment]
Yes,sir I need help

**Comment by utkarshnashine** [comment]
If you can point me to the resources

**Comment by praveenscience** [comment]
@utkarshnashine Happy to help you mate. What do you wanna know?

**Comment by utkarshnashine** [comment]
Sir shall I first go through through rezume data structure then ask?

**Comment by praveenscience** [comment]
Great, please do that @utkarshnashine

**Comment by utkarshnashine** [comment]
if I am understanding correctly then,empty arrays are already present in app.js and we need to implement forms so that we can extract data from them and put into the array right

**Comment by praveenscience** [comment]
@utkarshnashine Almost right, but what kind of data is what we are discussing here. 😁

**Comment by utkarshnashine** [comment]
I think I am wrong,but hobbies will data in the form of text only but we can create an object as suggested above and use them like for dancing there are different kinds of dacing styles like jazz, hip hop etc. as optional field and dancing as main field
              for eg:- Dancing
                                    Hip-Hop
                                     Jazz
like this

**Comment by praveenscience** [comment]
@utkarshnashine Seems a good idea. Let's improve upon it.

**Comment by utkarshnashine** [comment]
yes,sir I will more than happy to contribute so can I start working on it.
